### PR TITLE
S05 us00 style login signup

### PR DESCRIPTION
**Pull request by JohaneDecamps**

### CODE REVIEW

File: client/src/main.jsx [hunk 1]
```diff
@@ -51,7 +51,7 @@ const router = createBrowserRouter([
         element: <ResultPage />,
       },
       {
-        path: "/history9",
+        path: "/hystory9",
```
Comment (aurelienLavanchy) [inline client/src/main.jsx:54]: you mispelled `history`

Comment (JohaneDecamps) [reply client/src/main.jsx]: ✅

**Review by MichaelLELU — DISMISSED** [review]

**Review by aurelienLavanchy — DISMISSED** [review]

File: client/src/pages/loginpage/LoginPage.jsx [hunk 2]
```diff
@@ -21,77 +21,98 @@ export default function SignupPage() {
...
-        .post(`${expressURL}/api/auth/login`, data, 
+        .post(
+          `${expressURL}/api/auth/login`,
+          data,
           {
-          withCredentials: true,
-        },
+            withCredentials: true,
+          },
           {
```
Comment (ioayoub) [inline client/src/pages/loginpage/LoginPage.jsx:30]: ?

**Review by ioayoub — CHANGES_REQUESTED** [review]

File: client/src/pages/loginpage/LoginPage.jsx [hunk 3]
```diff
@@ -21,77 +21,98 @@ export default function SignupPage() {
...
-          headers: { 'Content-Type' : 'application/x-www-form-urlencoded; charset=UTF-8' },
-        })
+            headers: {
+              "Content-Type":
+                "application/x-www-form-urlencoded; charset=UTF-8",
```
Comment (ioayoub) [inline client/src/pages/loginpage/LoginPage.jsx:33]: ?

**Review by ioayoub — CHANGES_REQUESTED** [review]

**Review by aurelienLavanchy — APPROVED** [review]

**Review by MichaelLELU — APPROVED** [review]

**Review by aurelienLavanchy — APPROVED** [review]

**Review by MichaelLELU — DISMISSED** [review]

**Review by aurelienLavanchy — APPROVED** [review]

**Review by MichaelLELU — APPROVED** [review]

**Review by ioayoub — APPROVED** [review]

**Pull request merged**